vault/logical_system.go

```diff
@@ -1717,6 +1733,118 @@ func (b *SystemBackend) handleTuneWriteCommon(ctx context.Context, path string,
 		}
 	}
 
+	// user-lockout config
+	{
+		var apiuserLockoutConfig APIUserLockoutConfig
+
+		userLockoutConfigMap := data.Get("user_lockout_config").(map[string]interface{})
+		var err error
+		// Augmenting userLockoutConfigMap for some config options to treat them as comma separated entries
+		if userLockoutConfigMap != nil && len(userLockoutConfigMap) != 0 {
```

Comment:
I believe if we first check for nil `userLockoutConfigMap` ( possibly with addressing the above comment), we could avoid on layer of indentation with this if. Then, it is more readable.  What do you think ?

Reply:
I followed how we check for mount config in handleMount. I think the reason we do not check for nil first is that if this value is nil, we do not want to return an error and we are only concerned about "user_lockout_config"  only when it has values for it.